vscode-extension/src/util.ts

```diff
@@ -381,6 +381,26 @@ export async function setupEnvironmentVariables(
   }
 }
 
+export function validateConfigName(name: string, mode: "json" | "yaml") {
+  if (name === "") {
+    return "Name is required";
+  }
+  if (name.includes(" ")) {
+    return "Name cannot contain spaces";
```

Comment:
Why is this an issue?

Reply:
I'll test it out, but just assumed having spaces in the file name would be problematic as we've seen in other places

Reply:
Tested it out, made a config with spaces and it ran and saved fine